<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttachmentDialog</class>
 <widget class="QDialog" name="AttachmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Attachment</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>242</y>
     <width>200</width>
     <height>42</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>390</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>120</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Caption</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="captionEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>460</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
